runtime/shibuya/src/lib.rs

```diff
@@ -249,7 +249,9 @@ parameter_types! {
     pub RuntimeBlockWeights: BlockWeights = BlockWeights::builder()
         .base_block(BlockExecutionWeight::get())
         .for_class(DispatchClass::all(), |weights| {
-            weights.base_extrinsic = ExtrinsicBaseWeight::get();
+            // Adjusting the base extrinsic weight to account for the additional database
+            // read introduced by the `tx-pause` pallet during extrinsic filtering.
+            weights.base_extrinsic = ExtrinsicBaseWeight::get().saturating_add(RocksDbWeight::get().reads(1));
```

Comment:
This isn't the way to do it 🙂 

* `RocksDb` weights is a config param of the `frame_system` and we should use it from there
* That aside, this shouldn't be necessarily hardcoded but it should be benchmarked. There is a command to measure base weight and perhaps now is the time to use it.

Reply:
Do you mean using the `/bench` command from the workflows? Should I run it specifically for `frame-system` and `pallet-tx-pause`, or should I include others as well? I’m thinking of pallets that involve dispatches, such as **proxy, collective-proxy, utility, and scheduler**.

Reply:
There is a specific command to measure _overhead_, or base extrinsic weight.

E.g. using the frame-omni-bencher, it should be `frame-omni-bencher v1 benchmark overhead`.

I guess we'd need to adapt the CI a bit 🤔

Reply:
Thanks for pointing out this command to me; I was not aware of its existence! The overhead command is not yet available in `frame-omni-bencher` **v0.7.0** but should be released in the next Polkadot SDK **stable2412** version. I've adjusted the CI to include it alongside our existing benchmarking approach. I will run it below using the `/bench` command.

Reply:
Nice, very neat!